src/mgxs_data.F90

```diff
@@ -62,23 +61,6 @@ subroutine read_mgxs()
     allocate(micro_xs(n_nuclides_total))
 !$omp end parallel
 
-    ! Find out if we need fission & kappa fission
```

Comment:
I removed the `get_kfiss` and `get_fiss` so that the all xs are initialized whether the user input them or not. If fission and kappa-fission are not set, they will be ZERO.